ci/labels/package.json

```
@@ -0,0 +1,9 @@
+{
+  "private": true,
+	"type": "module",
```

Looks like the formatting is messed up here 


```suggestion
  "type": "module",
```